### PR TITLE
Adding Action to convert all guides at once

### PR DESCRIPTION
**Pull request by jakub-pomykala**
as requested here is the action to run all. the guides at once.

### CODE REVIEW

File: .github/workflows/RunConverterOnAllGuides.yml [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+jobs:
+  convertProd:
+    name: Make PR to Prod
+    runs-on: ubuntu-latest
+    if: contains(github.event.inputs.branch, 'master') || contains(github.event.inputs.branch, 'main') || contains(github.event.inputs.branch, 'prod')
+    strategy:
+      matrix: # Uses an array of Json variables to pass the repo names.
+              # The names differ between Github and Gitlab so this is necessary.
+              # Add new cloud-hosted-guides here to add them to the mirror process.
+              # i.e. {"github":"new-lab-github-url","gitlab":"new-lab-gitlab-url"}
```
Comment (austin0) [inline .github/workflows/RunConverterOnAllGuides.yml]: Comment needs an update.

File: .github/workflows/RunConverterOnAllGuides.yml [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+          git config --global user.name "GuidesBot"
+          git commit -m "Updated by github actions from ALL Guides"
+          git checkout --ours instructions/cloud-hosted-${{matrix.guide.guideName}}/instructions.md
+    
+      - uses: peter-evans/create-pull-request@v3
+        with:
+            title: "To Prod: Change to ${{matrix.guide.guideName}}"
+            token: ${{ secrets.GUIDECONVERSIONTOOL_PASSWORD }}
+            body: "Updated by github actions, triggered by the MANUAL ACTION"
+            branch: "cloud-hosted-${{matrix.guide.guideName}}/Prod"
```
Comment (austin0) [inline .github/workflows/RunConverterOnAllGuides.yml]: Prod -> prod

File: .github/workflows/RunConverterOnAllGuides.yml [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+
+  convertStaging:
+    name: Make PR to Staging
+    runs-on: ubuntu-latest
+    if: contains(github.event.inputs.branch, 'qa') || contains(github.event.inputs.branch, 'staging')
+    strategy:
+      matrix: # Uses an array of Json variables to pass the repo names.
+              # The names differ between Github and Gitlab so this is necessary.
+              # Add new cloud-hosted-guides here to add them to the mirror process.
+              # i.e. {"github":"new-lab-github-url","gitlab":"new-lab-gitlab-url"}
```
Comment (austin0) [inline .github/workflows/RunConverterOnAllGuides.yml]: Same comment update.

**Review by austin0 — APPROVED** [review]
Some minor changes

File: .github/workflows/RunConverterOnAllGuides.yml (hunk 2 above)
Comment (jakub-pomykala) [reply .github/workflows/RunConverterOnAllGuides.yml]: Done

File: .github/workflows/RunConverterOnAllGuides.yml (hunk 1 above)
Comment (jakub-pomykala) [reply .github/workflows/RunConverterOnAllGuides.yml]: Done

File: .github/workflows/RunConverterOnAllGuides.yml (hunk 3 above)
Comment (jakub-pomykala) [reply .github/workflows/RunConverterOnAllGuides.yml]: Done

**Review by gkwan-ibm — APPROVED** [review]

**Pull request merged**